src/components/common/NotesTable/index.js

```diff
@@ -14,49 +14,87 @@ import {
 import { columns } from './NoteUtils';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
 import { connect } from 'react-redux';
+import { useLocation } from 'react-router-dom';
 import '../styles/Notes.css';
+// edit comment ant framework
+const Editor = ({ onChange, onSubmit, submitting, onCancel, value }) => (
+  <>
+    <Form.Item>
+      <Input.TextArea rows={4} onChange={onChange} defaultValue={value} />
+    </Form.Item>
+    <Form.Item>
+      <Button
+        htmlType="submit"
+        loading={submitting}
+        onClick={onSubmit}
+        type="primary"
+        style={{}}
+      >
+        Save edit
+      </Button>
+      <Button onClick={onCancel}>Cancel</Button>
+    </Form.Item>
+  </>
+);
 
-const NotesTable = ({ userProfile }) => {
+const NotesTable = ({ userProfile, accounts }) => {
   const [data, setData] = useState([]);
+  let result;
   // edit users own comment states
   const [editing, setEditing] = useState(false);
-  const [editNote, setEditNote] = useState({ key: '', note: '' });
+  const [editNote, setEditNote] = useState({ key: '', content: '' });
   const [submitting, setSubmitting] = useState(false);
   // Get profile_id of logged in user
   const { profile_id } = userProfile;
-  const { TextArea } = Input;
-
+  const location = useLocation();
   // Dummy data for table
   useEffect(() => {
     axiosWithAuth()
-      .get('https://mocki.io/v1/cc34de61-aaf5-4725-b0c9-6d67efa3aff3')
+      .get(
+        location.pathname === '/notes' || '/mynotes'
+          ? '/notes'
+          : `/notes/mentees/${accounts.key}`
+      )
       .then(res => {
-        setData(res.data);
+        console.log(res);
+        setData(
+          res.data.map(obj => {
+            let created = new Date(obj.created_at);
+            return {
+              ...obj,
+              key: obj.note_id,
+              date: created.toDateString(),
+              time: created.toLocaleTimeString(),
+            };
+          })
+        );
       })
       .catch(err => {
-        console.log(err);
+        console.log(err.message);
       });
-  }, []);
+  }, [editing]);
 
+  if (location.pathname === '/mynotes') {
+    result = data.filter(
+      x => x.mentor_id === profile_id || x.mentee_id === profile_id
+    );
+  } else {
+    result = data;
+  }
   // click handlers
-  const handleMenuClick = e => {
-    // menu click handler
-  };
-
-  const handleDropDownClick = e => {
-    // dropdown click handler
-  };
-
-  const toggle = (key, note) => {
-    //editing note is currently for all comments
+  const handleMenuClick = e => console.log('click', e);
```

Comment:
I am guessing this will be needed in the future and that is why the console.log was kept. If it is not these could be deleted.